config/express.js

```diff
@@ -2,7 +2,7 @@
 
 const assert = require("assert");
 const express = require("express");
-const cors = require("cors");
+// const cors = require("cors");
```

Comment:
The cors might be used in production as during testing you cant use it. The chatbot client component (wip) send his cookie over the network. With the 'access-control-Allow-origin = * ', the XHR request won't work as it is "insecure". Depending on the infrastructure used, i'll change it but currently i have no solution for this issue.